python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -366,13 +366,27 @@ Returns whether to display labels on terrain tiles
 Returns list of point lights defined in the scene
 
 .. versionadded:: 3.6
+%End
+
+    QList<QgsDirectionalLightSettings> directionalLights() const;
+%Docstring
+Returns list of directional lights defined in the scene
+
+.. versionadded:: 3.16
 %End
 
     void setPointLights( const QList<QgsPointLightSettings> &pointLights );
 %Docstring
 Sets list of point lights defined in the scene
 
 .. versionadded:: 3.6
+%End
+
+    void setDirectionalLights( const QList<QgsDirectionalLightSettings> &pointLights );
```

Comment:
For some reason here it's still `pointLights` although in the header it correctly says `directionalLights`